tests/lean/run/bv_decide_rewriter.lean

```diff
@@ -88,6 +88,14 @@ example {x : BitVec 16} : x / (BitVec.ofNat 16 8) = x >>> 3 := by bv_normalize
 example {x y : Bool} (h1 : x && y) : x || y := by bv_normalize
 example (a b c: Bool) : (if a then b else c) = (if !a then c else b) := by bv_normalize
 
+-- neg_mul' / mul_neg'
+example (x y : BitVec 16) : (-x) * y = -(x * y) := by bv_normalize
```

Comment:
It seemed useful to me to have both the normalized and unnormalized forms, but happy to reduce this to just the normalized if you prefer.

Reply:
I think this is fine but please also add test cases for the symmetric variants mentioned above.